### DISCUSSION TITLE
Add Dataobject Button does not work

### DISCUSSION THREAD

**Discussion by BjBue** [pimcore/pimcore#14764] [Q&A / Support] [answered]
Hello,

how is it possible that the Add-button under "settings->dataobjects->classes" does not work?
It does not work on staging and it does not work in production. 
Is this a feature? And how can i make it working again?
I am very confused because it is the default way to create data objects, isn't it?
Best regards

**Comment by dvesh3** [comment, marked as answer]
Hi @BjBue you might have `PIMCORE_CLASS_DEFINITION_WRITABLE` env var set to 0 on production which doesn't allow to create new classes. read more about the deployment config [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Deployment_Tools.html#page_Pimcore-Class-Definitions)
